### PR TITLE
Replace PRM loading spinner

### PR DESCRIPTION
**Pull request by strempe**
## What's this PR do?
Replaces loading spinner in PRM with the same spinner used in React apps.

## Screenshots (if appropriate)
screen shot 2017-10-12 at 4 15 05 pm

## Where should the reviewer start?
PRM Partners

## How should this be manually tested?
Each page in PRM should show the React spinner when ajax-busy is deselected for "display: none;". 

### Front-End

- [ ] Works on IE 11 and Edge
- [ ] Works on Mobile
- [ ] Unit tests added or ticket exists to add tests

### Separation of Concerns

- [ ] Back end "business logic" is implemented in plain python, Django model classes, and Django forms. (not Django views or other frameworks)
- [ ] Concerns of UI, api, and model are in separate code units.

### Tests

- [ ] Base classes used for python tests don't do too much unnecessary work.

### Migrations

- [ ] Migrations can be reversed.
- [ ] Current production code would continue to work after this migration is applied.
- [ ] Data migrations' forward/reverse functions should be decorated with `@only_on_default`.

### CODE REVIEW

**Comment by tdphillips** [comment]
Our React spinner dims the page, doesn't it?

**Comment by strempe** [comment]
@tdphillips Yes, I will add the overlay.

**Comment by aljordan82** [comment]
@strempe There is a LoadingSpinner component that we use in other apps. can't it be used here?

**Comment by mwarnsleyDE** [comment]
@strempe I do believe it's the component inside of the common/ui directory in the gulp folder for the LoadingSpinner component that we can just prop in there

**Comment by strempe** [comment]
@aljordan82 @mwarnsleyDE There is, I'll try that. I was under the impression it wouldn't work in Django.

**Comment by aljordan82** [comment]
@strempe LoadingSpinner component will not work in django.

**Comment by mwarnsleyDE** [comment]
Man this switching between Django and React is confusing lol. I was under the impression that PRM was a React application isn't it? I'm not sure how much of the rest of it is normal jQuery or what.

**Comment by koonsies** [comment]
PRM is a Django app. Any Javascript is regular JS/JQuery.

**Comment by koonsies** [comment]
As a future note, is it possible to add a comment to files that have been changed by Prettier, but do not contain actual code changes? Even with `?w=1`, a lot of formatting changes clutter the PR and make it take a bit longer to read. If there is a file in a PR that contains -only- formatting changes, you can just add a comment at the top saying "Formatting only" or something along those lines.

**Comment by tdphillips** [comment]
Note, that should be a GitHub comment not a js/css comment.

File: static/my.jobs.css [hunk 1]
```diff
@@ -2252,10 +2252,9 @@ table#row-link-table {
     width: 400px;
     height: 150px;
     text-align: center;
-    background: #FFF url(//d2e48ltfsb5exy.cloudfront.net/content_mj/files/images/ajax-loader.gif) no-repeat center center;
-    opacity: .75;
-    border: 1px solid #000;
     z-index: 9999;
+    top: 400px;
```
Comment (aljordan82) [inline static/my.jobs.css]: Arbitrary values! this will not work on different screen sizes.

File: templates/base-react.html [hunk 2]
```diff
@@ -153,7 +157,9 @@
         </footer>
         <script>
             var staticUrl = "{% static "" %}";
-            var spinnerImg = "{% static 'images/ajax-loader.gif' %}";
+            <div class="big-chart big-trans-box" id="ajax-loading-donut">
+                <div class="loader"></div>
```
Comment (mwarnsleyDE) [inline templates/base-react.html]: Does this part need the overlay as well?

File: gulp/src/sass/custom.scss [hunk 3]
```diff
@@ -2545,10 +2545,9 @@ table#row-link-table {
   width: 400px;
   height: 150px;
   text-align: center;
-  background: #FFF url("//d2e48ltfsb5exy.cloudfront.net/content_mj/files/images/ajax-loader.gif") no-repeat center center;
-  opacity: .75;
-  border: 1px solid #000;
   z-index: 9999;
+  top: 400px;
```
Comment (mwarnsleyDE) [inline gulp/src/sass/custom.scss]: May need a percentage for the top value for responsiveness

File: templates/analytics/analytics_main.html [hunk 4]
```diff
@@ -37,7 +37,9 @@
 {% block section %}
     <section id="analytics_content">
         <div id="content" data-analyticsType="{{ analytics_type }}"></div>
-        <div id="ajax-busy"></div>
+        <div className="big-chart big-trans-box" id="ajax-busy">
+            <div className="loader"></div>
```
Comment (mwarnsleyDE) [inline templates/analytics/analytics_main.html]: Does this portion need the overlay as well?

**Pull request merged**